src/types/index.ts

```diff
@@ -172,7 +172,7 @@ export interface QuestionOption {
  * Type describing a question with text and options in New Quiz Screen
  */
 export interface Question {
-  questionText: string;
+  question: string;
```

Comment:
Was there a reason to change from questionText to question. As the interface is called Question it seems reasonable to me to call the attribute questionText